src/dex/balancer-v2/balancer-v2-e2e.test.ts

```diff
@@ -20,6 +21,152 @@ describe('BalancerV2 E2E', () => {
       network,
     );
 
+    describe('Popular tokens', () => {
```

Comment:
Why don't we have test cases for BUY side on other networks (Arbitrum, Polygon etc) as well?

Reply:
@skypper Tests for polygon and arbitrum were added to test Weighted Pools